File: AkitaModel/SmartScan/File.cs
namespace Skyware.Lis.AkitaModel.SmartScan;

/// <summary>
/// Binary file with metadata.
/// </summary>
public class File : FileBase
{

    /// <summary>
    /// Base64 encoded binary data of a file.
    /// </summary>
    public string Data { get; set; }

}

File: AkitaMotelTS/src/SmartScan/File.ts
/**
 * Skyware.Lis.AkitaModel.SmartScan
 */

import { FileBase } from './FileBase';

/**
 * Binary file with metadata.
 */
export class File extends FileBase {
  /**
   * Base64 encoded binary data of a file.
   */
  data?: string;
}
